package.json

```diff
@@ -76,6 +89,7 @@
         "agentkeepalive": "4.5.0",
         "amqplib": "^0.10.4",
         "axios": "^1.7.2",
+        "c8": "^10.1.2",
```

Comment:
this should be a dev dependency. pls also make sure to re-run `npm i` to bring that to the lockfile.